### ISSUE TITLE
Feature request: Installing plugins from HTTP-source

### ISSUE THREAD

**Issue by kvaps** [kubernetes-sigs/krew#400]
Hi, since main [krew-index](https://github.com/kubernetes-sigs/krew-index) repository have quite strict moderation, I'd like to host manifests for my plugins on my own and provide krew users simple command to install it like:

```
kubectl krew install --manifest=https://github.com/kubernetes-sigs/krew-index/raw/f05fb4e8caff684954df122dc356d532163c0e85/plugins/use.yaml
```

Also it would be nice allow to use  `-f` option instead of `--manifest` because this is quite common for kubectl command

**Comment by corneliusweig** [comment]
You will be glad to learn that krew has a multi-index feature on its roadmap. However, it will take some time until it's here.

Your current approach to install directly from manifest URL has been implemented here #350, but the work is not quite finished yet. Installing via a manifest reference has one important drawback, namely that users will not receive updates.

Consequently, the `--manifest` option is just intended for development purposes. In that light, a shorthand option `-f` does not make sense to me.

**Comment by ahmetb** [comment]
> since main [krew-index](https://github.com/kubernetes-sigs/krew-index) repository have quite strict moderation

For this we'll support tapping to custom plugin indexes (#23). So installing plugin manifests from HTTP URL is not really what you need.
/close
